node/Cargo.toml

```diff
@@ -16,19 +16,25 @@ path = "src/main.rs"
 [dependencies]
 # Frequency Runtime
 frequency-cli = {path = "cli"}
-frequency-rococo-runtime = {path = "../runtime/frequency-rococo"}
-frequency-runtime = {path = "../runtime/frequency"}
-
+frequency-rococo-runtime = {path = "../runtime/frequency-rococo", optional = true}
+frequency-runtime = {path = "../runtime/frequency", optional = true}
+frequency-service = {path = "service"}
 # Substrate
 sc-cli = {git = "https://github.com/paritytech/substrate", features = ["wasmtime"], branch = "polkadot-v0.9.27" }
 
 [build-dependencies]
 substrate-build-script-utils = {git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.27" }
 
 [features]
+std = []
+default = ["std", "all-frequency-features"]
 runtime-benchmarks = [
   "frequency-runtime/runtime-benchmarks",
   "frequency-rococo-runtime/runtime-benchmarks",
   "frequency-cli/runtime-benchmarks",
 ]
+frequency = ["frequency-service/frequency", "frequency-cli/frequency"]
+frequency-rococo-local = ["frequency-service/frequency-rococo-local", "frequency-cli/frequency-rococo-local"]
+frequency-rococo-testnet = ["frequency-service/frequency-rococo-testnet", "frequency-cli/frequency-rococo-testnet"]
+all-frequency-features =["frequency-service/all-frequency-features", "frequency-cli/all-frequency-features"]
 try-runtime = ["frequency-runtime/try-runtime", "frequency-rococo-runtime/try-runtime"]
```

Comment:
define 3 features
1. frequency  = mainnet
2. frequency-rococo-testsnet = rococo testnet features 
3. frequency-rococo-local = same as frequency-rococo-test + instant sealing is only included in this feature